_drafts/semantic-release.md

```diff
@@ -0,0 +1,37 @@
+---
+state: In Progress
+---
+
+# Set up release workflow with `semantic-release`
+
+`semantic-release` makes releasing faster, simpler and safer. All of these are valuable if we want to increase public participation in the project and increase momentum and motivation.
```

Comment:
I see there's links to semantic-release further down, but could you make sure there's a link the first time it's mentioned?